images/kcp-registrar/register.sh

```diff
@@ -20,54 +20,58 @@ set -o pipefail
 
 usage() {
 
-    printf "Usage: KCP_ORG="root:pipelines-service" KCP_WORKSPACE="compute" DATA_DIR="/workspace" ./register.sh\n\n"
+    printf "Usage: KCP_ORG="root:pipeline-svc" KCP_WORKSPACE="compute" DATA_DIR="/workspace" ./register.sh\n\n"
```

Comment:
sorry but I don't see the rational for changing that. Readability is in my opinion more important than saving the typing of 4 characters, especially when tab completion allows not to type them.

Reply:
I changed it because our current org workspace is named pipeline-svc. If you don't agree I can revert that change, it is not a big deal.